client/app/api/course/Survey/Responses.js

```diff
@@ -2,19 +2,27 @@ import BaseSurveyAPI from './Base';
 
 export default class ResponsesAPI extends BaseSurveyAPI {
   /**
-  * answer_with_response = {
+  * survey_response = {
   *   survey: {
   *     id:number, title:string, description:string, start_at:datetime, ...etc,
   *       - Survey attributes
-  *     questions: Array.<{description:string, options:Array, question_type:string... etc}>,
-  *        - Array of questions belonging to the survey
-  *        - question_type is one of ['text', 'multiple_choice', 'multiple_response']
   *   },
   *   response: {
   *     id:number, submitted_at:datetime,
   *       - Response Attributes
-  *     answers: Array.<question_id:number, text_response:string, options:Array. etc...>,
-  *       - Answer attributes
+  *     sections:
+  *       Array.<
```

Comment:
```js
Array.<{
   id: number, 
}>
```

Missing `{` after `Array`, `{}` indicates it's an object. And let's add a space before the types (number/string)? , I am not sure why some of my comments also missing the spaces, will add them later.